### ISSUE TITLE
Needs HTTPS

### ISSUE THREAD

**Issue by stjohnjohnson** [jer/justaforkbomb.com#1]
I don't feel comfortable using this service as it doesn't provide TLS.  Can you add it please?

**Comment by jer** [comment]
I totally agree. The problem is that this is a Heroku app, and TLS+Custom domain+Heroku := $

I'm open to ideas.

**Comment by FenrirUnbound** [comment]
Host it on Google's App Engine. IIRC free HTTPS hosting

**Comment by jer** [comment]
Is it possible to use LetsEncrypt with auto-renew from App Engine?
